components/renku_data_services/k8s/config.py

```diff
@@ -0,0 +1,111 @@
+"""Base config for k8s."""
+
+import glob
+import os
+
+import kr8s
+import yaml
+from sanic.log import logger
+
+from renku_data_services.k8s.constants import DEFAULT_K8S_CLUSTER
+from renku_data_services.k8s.models import Cluster, ClusterId
+
+
+class KubeConfig:
+    """Wrapper around kub config to get a kr8s api."""
+
+    def __init__(
+        self,
+        kubeconfig: str | None = None,
+        current_context_name: str | None = None,
+        ns: str | None = None,
+        sa: str | None = None,
+        url: str | None = None,
+    ) -> None:
+        self._kubeconfig = kubeconfig
+        self._ns = ns
+        self._current_context_name = current_context_name
+        self._sc = sa
```

Comment:
suggestion: This is a leftover from Lionel's PR. I think he misunderstood my comment. We should rename `self._sc` to `self._sa` because this is the service account. And the short name for service account is `sa` not `sc`. `sc` is actually the short name for storage class.

Reply:
could also be a rebase mistake. I'll fix it

Reply:
That is a good point. Thanks!